lib/src/json_schema/json_schema.dart

```diff
@@ -1018,62 +1018,141 @@ class JsonSchema {
       'contentEncoding': (JsonSchema s, dynamic v) => s._setContentEncoding(v),
     });
 
-  static Map<String, SchemaPropertySetter> _accessMapV2019_09 = Map<String, SchemaPropertySetter>()
-    ..addAll(_baseAccessMap)
-    ..addAll(_accessMapV7)
-    // "Note that the standard meta-schema still reserves definitions for backwards compatibility"
-    // https://json-schema.org/draft/2019-09/release-notes.html#core-vocabulary
-    // https://json-schema.org/draft/2019-09/json-schema-core.html#rfc.section.8.2.5
-    // Unfortunately the spec does not explicitly say to ignore these old keywords,
-    // nor do the spec tests offer an answer. However, the sane thing to do seems to be to ignore them,
-    // otherwise we get into undocumented territory.
-    ..remove(r'$definitions')
-    ..remove(r'$dependencies')
-    ..addAll({
-      // Note: see https://json-schema.org/draft/2019-09/release-notes.html
+  // TODO: Think about doing something with the deprecated keywords in this map.
+  static Map<String, SchemaPropertySetter> _draft2019_combined = Map()
+    ..addAll(_draft2019_core)
+    ..addAll(_draft2019_applicator)
+    ..addAll(_draft2019_content)
+    ..addAll(_draft2019_format)
+    ..addAll(_draft2019_metadata)
+    ..addAll(_draft2019_validation);
 
-      // Added or changed in draft2019_09: Core Vocabulary
+  static Map<String, SchemaPropertySetter> _draft2019_core = Map<String, SchemaPropertySetter>()
```

Comment:
It's actually really nice to have a concise synopsis of what is in each vocabulary!